template/lambda-sqs-worker-cdk/skuba.template.js

```diff
@@ -22,6 +22,18 @@ module.exports = {
       initial: 'my-team-aws-account-prod:cicd',
       validate: (value) => /^.+:.+$/.test(value),
     },
+    {
+      name: 'devDataDogApiKeySecretArn',
+      message: 'Dev DataDog API Key Secret ARN',
+      initial:
+        'arn:aws:secretsmanager:<Region>:<AccountId>:secret:<SecretName>',
```

Comment:
1. Is this typically a different ARN between dev and prod?
2. Are our accounts provisioned with a standard secret name by default, and if so, could we use that as an actual default here?

Reply:
1. I think most teams have a different ARN as they haven't facilitated the sharing mechanism between accounts
2. Looks like `datadog-api-key` is pretty standard